http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcTest.java

```diff
@@ -496,6 +605,50 @@ private void performTenantRequest(String username, String password, String tenan
         }
     }
 
+    private void testRequestObjectInvalidConfiguration(InputStream oidcConfig, RequestObjectErrorType requestObjectErrorType) throws Exception {
+        try {
+            Map<String, Object> props = new HashMap<>();
+            try {
+
+                OidcClientConfiguration oidcClientConfiguration = OidcClientConfigurationBuilder.build(oidcConfig);
+                assertEquals(OidcClientConfiguration.RelativeUrlsUsed.NEVER, oidcClientConfiguration.getRelativeUrls());
+
+                OidcClientContext oidcClientContext = new OidcClientContext(oidcClientConfiguration);
+                oidcFactory = new OidcMechanismFactory(oidcClientContext);
+                HttpServerAuthenticationMechanism mechanism;
+
+                if (oidcClientConfiguration.getAuthenticationRequestFormat().contains(REQUEST.getValue())) {
+                    mechanism = oidcFactory.createAuthenticationMechanism(OIDC_NAME, props, getCallbackHandler(true, "+phone+profile+email"));
+                } else {
+                    mechanism = oidcFactory.createAuthenticationMechanism(OIDC_NAME, props, getCallbackHandler());
+                }
+
+                URI requestUri = new URI(getClientUrl());
+                TestingHttpServerRequest request = new TestingHttpServerRequest(null, requestUri);
+                try {
+                    mechanism.evaluateRequest(request);
+                } catch (Exception e) {
```

Comment:
What happens if when running the test, the expected exception does not occur? It looks like the test would pass erroneously in this case.

After `mechanism.evaluateRequest`, you could call
`Assert.fail("No error was thrown while attempting to evaluate the request");`  to make sure that the test will fail if evaluateRequest does not result in an error as expected.

Reply:
Can we add an assertion to check the status code and result like for the wrong password case here:

https://github.com/wildfly-security/wildfly-elytron/blob/f77ddaa8e4b292961e67387c7378f6e0932016b6/http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcTest.java#L135C5-L136C59

Reply:
Ah, right, the wrong password case is different in that no exception is thrown there. No need to add the status code check then but we should make sure the test fails if evaluateRequest doesn't throw an exception.

Reply:
Sounds good! 
I have added the assert.fail() statement and changed the names of the tests a bit. 
Thanks @fjuma